src/handlers/web3.ts

```diff
@@ -721,31 +742,7 @@ export const getDataForNftTransfer = async (
       convertStringToHex('0'),
     ]);
     return data;
-  } else if (IS_TESTING === 'true') {
-    const transferMethodHash = smartContractMethods.nft_transfer.hash;
-    const data = ethereumUtils.getDataString(transferMethodHash, [
-      ethereumUtils.removeHexPrefix(to),
-      convertStringToHex(asset.id),
-    ]);
-    return data;
   }
-
-  const address = asset.asset_contract?.address!;
-  const abi = await fetchContractABI(address);
-  const iface = new Interface(abi);
-
-  const isTransferFrom =
-    iface.functions?.[smartContractMethods.nft_transfer_from.method];
-  const transferMethodHash = isTransferFrom
-    ? smartContractMethods.nft_transfer_from.hash
-    : smartContractMethods.nft_transfer.hash;
-
-  const data = ethereumUtils.getDataString(transferMethodHash, [
-    ...(isTransferFrom ? [ethereumUtils.removeHexPrefix(from)] : []),
-    ethereumUtils.removeHexPrefix(to),
-    convertStringToHex(asset.id),
-  ]);
-  return data;
 };
```

Comment:
we should have a fallback default case which prob should be a normal 721 send 

id check ->  if 1155 handle, if 721 handle, if neither, add reporting with network, contract address, name of collection and then optimistically try a 721 scenario